### DISCUSSION TITLE
cURL Error on Specific Font

### DISCUSSION THREAD

**Discussion by cjmellor** [spatie/laravel-google-fonts#6] [Q&A] [answered]
Hi,

I'm trying the package for the first time today.

I've added these three options in the config

```php
'fonts' => [
    'anton' => 'https://fonts.googleapis.com/css2?family=Anton&display=swap',
    'merriweather' => 'https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap',
    'roboto-mono' => 'https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap',
],
```

and when I run `php artisan google-fonts:fetch` it successfully pulls in the first two listed, but when it gets to `roboto-mono`, it fails. When I hit the URL manually, it connects fine.

### Error produced

```
cURL error 6: Could not resolve host: fonts.gstatic.com (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://fonts.gstatic.com/s/robotomono/v13/L0x5DF4xlVMF-BfR8bXMIjhPq3-cXbKDO1w.woff2 (View: /Users/chris/Dev/Private/mellor.pizza/resources/views/components/layout.blade.php)
```

Thanks.

**Comment by EricBusch** [comment, marked as answer]
I was also getting the `cURL error 6: Could not resolve host: fonts.gstatic.com` when running locally using HTTP and fixed it by removing `https` from the fonts.googleapis.com URLs like this:

```php
'fonts' => [
    'anton'        => '//fonts.googleapis.com/css2?family=Anton&display=swap',
    'merriweather' => '//fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap',
    'roboto-mono'  => '//fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap',
],
```

Hope this helps!

**Reply by cjmellor** [reply]
Thanks @EricBusch 

Changing to `http` made it work.

**Reply by cjmellor** [reply]
To be fair, I think I spoke too soon on this unless I'm missing something...

It builds up the `.css` file (which it didn't before) but doesn't generate the font files.

**Reply by blakeyman** [reply]
I get this error if I change to "http"

```Blocked loading mixed active content "http://fonts.gstatic.com/s/jetbrainsmono/v6/tDbV2o-flEEny0FZhsfKu5WU4xD7OwGtT0rU.woff2"```

**Reply by EricBusch** [reply]
Try removing `http` and `https` from the URLs. I think something like this should solve the issue:

```php
'fonts' => [
    'anton'  => '//fonts.googleapis.com/css2?family=Anton&display=swap',
],
```

**Reply by blakeyman** [reply]
Unfortunately not

Blocked loading mixed active content "http://fonts.gstatic.com/s/jetbrainsmono/v6/tDbv2o-flEEny0FZhsfKu5WU4zr3E_BX0PnT8RD8yKwBNntkaToggR7BYRbKPxDcwgknk-4.woff2 <http://fonts.gstatic.com/s/jetbrainsmono/v6/tDbv2o-flEEny0FZhsfKu5WU4zr3E_BX0PnT8RD8yKwBNntkaToggR7BYRbKPxDcwgknk-4.woff2>”

In config

`'fonts' => [
    'default' => '//fonts.googleapis.com/css2?family=JetBrains+Mono:ital,wght@0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600;1,700;1,800&display=swap',
],`
